torch/nn/functional.py

```diff
@@ -3512,6 +3512,60 @@ def multi_margin_loss(
 """,
 )
 
+_native_pixel_shuffle = _add_docstr(
+    torch._native_pixel_shuffle,
+    r"""
+_native_pixel_shuffle(input, upscale_factor) -> Tensor
+
+Native kernel level implementation of the `pixel_shuffle`.
+This function might become private in future releases, use with caution.
+
+Rearranges elements in a tensor of shape :math:`(*, C \times r^2, H, W)` to a
+tensor of shape :math:`(*, C, H \times r, W \times r)`, where r is the :attr:`upscale_factor`.
+
+See :class:`~torch.nn.PixelShuffle` for details.
+
+Args:
+    input (Tensor): the input tensor
+    upscale_factor (int): factor to increase spatial resolution by
+
+Examples::
+
+    >>> input = torch.randn(1, 9, 4, 4)
+    >>> output = torch.nn.functional._native_pixel_shuffle(input, 3)
+    >>> print(output.size())
+    torch.Size([1, 1, 12, 12])
+""",
+)
+
+_native_pixel_unshuffle = _add_docstr(
+    torch._native_pixel_unshuffle,
+    r"""
+_native_pixel_unshuffle(input, downscale_factor) -> Tensor
+
+Native kernel level implementation of the `pixel_unshuffle`.
+This function might become private in future releases, use with caution.
```

Comment:
doc nit: the function is private now since it begins with an underscore

Reply:
fixed